config/Shared/config_default.php

```diff
@@ -744,22 +748,37 @@
     OrderStatusChangedTransfer::class => 'orders',
     SearchEndpointAvailableTransfer::class => 'search',
     SearchEndpointRemovedTransfer::class => 'search',
+    ExportMerchantsTransfer::class => 'merchant-commands',
+    MerchantExportedTransfer::class => 'merchant-events',
+    MerchantCreatedTransfer::class => 'merchant-events',
+    MerchantUpdatedTransfer::class => 'merchant-events',
```

Comment:
@danielsantos-spryker would be good if this won't be changed in suite-nonsplit master in https://spryker.atlassian.net/browse/PBC-1682

otherwise, this PR has to wait or you will need to adjust it when you finish with your PR.

Reply:
@matweew they are not changed, check [here](https://github.com/spryker/suite-nonsplit/pull/7926/files#diff-63fd0eb4016cfe314f73fd68f8617c6bdb966353538807bf9b093df9f05a0eaeR863-R866)

Reply:
but the PR is still not merged ;) this is my main message

Reply:
No worries, it will be released soon and those channels are following the RFC since they were initially released.